snuba/admin/audit_log/base.py

```diff
@@ -31,7 +31,7 @@ def record(
             timestamp=timestamp,
             **data,
         )
+        blocks = build_blocks(data, action, timestamp, user)
```

Comment:
did this so that the code would get run even if the client wasn't configured (like in tests)